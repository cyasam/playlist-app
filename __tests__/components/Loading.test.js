import React from 'react'
import { shallow } from 'enzyme'
import Loading from '../../src/scripts/components/Loading'

describe('Loading Component', () => {
  const wrapper = shallow(<Loading />)

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('has loading class div', () => {
    expect(wrapper.find('.loading').exists()).toBe(true)
  })
})
