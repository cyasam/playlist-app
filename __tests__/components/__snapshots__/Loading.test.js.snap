// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component renders correctly 1`] = `
<div
  className="loading"
>
  <img
    alt="Loading..."
    src={null}
  />
</div>
`;
